05-06-25-assignment-4/question-2.js

```diff
@@ -0,0 +1,38 @@
+//2.Write a program to reverse a string ("Hello John" => "olleH nhoJ")
+
+const reversedString = (str)=>{
+    let n=str.length;
+    let splitString = str.split(" ");
+    let resString="";
+
+    for(let i=0;i<splitString.length;i++)
+    {
+      // let str1=splitString[0];
+       resString+=reverseWord(splitString[i])+" ";
+    }
+
+    return resString;
+
+}
+
+
+function reverseWord(word)
```

Comment:
Your whole program is too long and complex, please use string and array functions.